capabilities/test-automation.md

```diff
@@ -1,48 +1,78 @@
 # [Test Automation](https://dora.dev/devops-capabilities/technical/test-automation/)
 
-Test automation is essential for building quality into software by providing fast feedback throughout the software delivery lifecycle.
-It addresses the inefficiencies of manual testing, such as time consumption, unreliability, and delayed feedback.
-Automated testing should be integrated continuously, involving various types of tests, such as unit and acceptance tests, to ensure functionality and prevent regression errors.
+Test automation is essential for building quality into software.
+Manual testing is inefficient, time-consuming, and unreliable, leading to delayed feedback. Automated testing, on the other hand, provides fast feedback throughout the software delivery lifecycle. In order to ensure quality functionality and to prevent regression errors, test automation should be integrated continuously and should involve various types of tests such as unit and acceptance.
```

Comment:
This sentence feels a bit too dismissive of manual testing, which I view as an essential part of every testing plan. However, it's commonly over-relied upon, leading to a lot of the negative aspects outlined here. Can you revert the tone/framing to be more neutral?

Reply:
Reworked this in 11/14 commit.